tests/integration/users.test.js

```diff
@@ -146,43 +170,35 @@ describe('POST /users', () => {
     });
 
     it('should return a 403 if customer is not from the same company', async () => {
-      const payload = { ...userPayload, customers: [customerFromOtherCompany] };
-      const response = await app.inject({
-        method: 'POST',
-        url: '/users',
-        payload,
-        headers: { 'x-access-token': authToken },
-      });
-      expect(response.statusCode).toBe(403);
-    });
-
-    it('should return an error 409 if user already exist', async () => {
```

Comment:
Pourquoi avoir supprimé ce test ?

Reply:
car il y a deja ce test et j'ai l'impression que c;esst le meme `should not create a user if email provided already exists`

Reply:
bien vu !